src/Cardano/Address/Style/Shelley.hs

```diff
@@ -133,6 +140,12 @@ newtype Shelley (depth :: Depth) key = Shelley
 deriving instance (Functor (Shelley depth))
 instance (NFData key) => NFData (Shelley depth key)
 
+-- Unsafe constructor for easily lifting bytes inside an 'Shelley'.
+--
+-- /!\ Use at your own risks.
+unsafeMkShelleyKey :: key -> Shelley (depth :: Depth) key
+unsafeMkShelleyKey = Shelley
```

Comment:
Hmm. This feels very redundant with `liftXPrv`. Slightly more general though. What is the use-case exactly?

Reply:
I added `liftXPub` instead - https://github.com/input-output-hk/cardano-addresses/pull/31/commits/bec1b0678ea08813980e8d243a76b95be8d478bb